addons/website/static/src/snippets/s_popup/000.js

```diff
@@ -6,6 +6,37 @@ const dom = require('web.dom');
 const publicWidget = require('web.public.widget');
 const utils = require('web.utils');
 
+// TODO: In master return/export `SharedPopupWidget` too
+const SharedPopupWidget = publicWidget.Widget.extend({
+    selector: '.s_popup',
+    disabledInEditableMode: false,
+
+    /**
+     * @override
+     */
+    start() {
+        // A popup element is composed of a `.s_popup` parent containing the
+        // actual `.modal` BS modal. Our internal logic and events are hidding
+        // and showing this inner `.modal` modal element without considering its
+        // `.s_popup` parent. It means that when the `.modal` is hidden, its
+        // `.s_popup` parent is not touched and kept visible.
+        // It might looks like it's not an issue as it would just be an empty
+        // element (its only child is hidden) but it leads to some issues as for
+        // instance on chrome this div will have a forced `height` due to its
+        // `contenteditable=true` attribute. It will result in a ugly white bar.
+        // tl;dr: this is keeping those 2 elements visibility synchronized.
+        this.$target.on('shown.bs.modal.SharedPopupWidget', () => {
+            this.$target.show();
+        });
+        this.$target.on('hide.bs.modal.SharedPopupWidget', () => {
+            this.$target.hide();
+        });
```

Comment:
Probably buggy: you use the "shown" and "hide" events while it should probably be "show" and "hidden". My guess is that with "shown" and "hide", you don't see the opening/closing effects no more (since you only make the external div visible when the opening animation is finished and hide it before the closing animation starts).

Reply:
Damn, indeed :facepalm: